SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -27,14 +27,17 @@ class ViewController: UIViewController {
     @IBOutlet weak var callControlView: UIView!
     @IBOutlet weak var muteSwitch: UISwitch!
     @IBOutlet weak var speakerSwitch: UISwitch!
-
+    @IBOutlet weak var playMusicButton: UIButton!
+    @IBOutlet weak var callOptionsView: UIView!
+    @IBOutlet weak var customAudioDeviceSwitch: UISwitch!
+    
     var incomingPushCompletionCallback: (() -> Void)?
 
     var isSpinning: Bool
     var incomingAlertController: UIAlertController?
 
     var callKitCompletionCallback: ((Bool) -> Void)? = nil
-    var audioDevice = TVODefaultAudioDevice()
+    var audioDevice : TVOAudioDevice = TVODefaultAudioDevice()
```

Comment:
Previous version with type implied is more conventional. Am I missing something?

Reply:
@timrozum With the updates of this PR we want to let app users be able to choose between `TVODefaultAudioDevice` and custom audio device implemented based on the `TVOAudioDevice` interface.
The `TVODefaultAudioDevice` will be used by default.

What is the recommended way for this case?

Reply:
I don't think I realized TVOAudioDevice was a protocol, that makes great sense now.

I would suggest removing the space before the `:` for convention.